file_paths.md

```diff
@@ -121,14 +126,22 @@ The files annotated by an '*' indicate that they are created by package installa
         ssl *
         VERSION                           # puppet-agent package version
 
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            puppet *                      # executes Puppet and provides results of Puppet runs
```

Comment:
@ploubser that sounds ominous. I assume anything here that's a rename will require *some* work. Are you saying this one is notably or prohibitively sticky?